networksurvey/src/main/java/com/craxiom/networksurvey/services/NetworkSurveyService.java

```diff
@@ -1053,6 +1054,16 @@ private void setScanRateValues()
         updateLocationListener();
     }
 
+    /**
+     * @return The time duration, in milliseconds, during which we expect to receive at least one GNSS measurement
+     *
+     * @since 1.8.0
+     */
+    public static long getGnssTimeoutIntervalMs()
+    {
+        return gnssScanRateMs * 2L;
+    }
```

Comment:
I think we can get away from making the `gnssScanRateMs` static. The `gnssScanRateMs` is already in the `SurveyRecordProcessor`, and if you want to provide a single source of truth for calculating the GNSS timeout, then you could add a static method to a utils class, and pass in the `gnssScanRateMs` as a parameter. I think I like that more than changing the instance variable to a static. However, I might be missing a nuance here.